### ISSUE TITLE
virus dont download

### ISSUE THREAD

**Issue by Hexlin420** [C0MPL3XB/Discord-Token-Generator#18]
It grabs your discord token and send private messages to everyone

**Comment by Gingestttt** [comment]
hello , you prob got logged by another program

**Comment by QuartzWarrior** [comment]
You probably can't trust this as the [code](https://github.com/C0MPL3XB/Discord-Token-Generator/blob/main/main/util/ShuffleFunctions.pyw) is encrypted.

**Comment by Gingestttt** [comment]
well quartz i use it it works

**Comment by QuartzWarrior** [comment]
Yes, it works, but you should be careful.

**Comment by Gingestttt** [comment]
ok im using virtual machine anyways its not a virus

**Comment by misty00x** [comment]
This is true the .bat file you run, runs this Encrypted script: ShuffleFunctions.pyw
It should be easy enough to tell from these functions that it's causing harm to your system.
`exec:
  self._bytes,self._exec,_delete,_AES256[_exit],self._eval,self._system=lambda _AES256:self._exec(_delete(_AES256))`
  
  If you're not alarmed by the words 'system' and 'delete' in the same string.. oh boy...

**Comment by QuartzWarrior** [comment]
Yup

**Comment by QuartzWarrior** [comment]
I'll try to make a new version of this, without the encrypted code.
